packages/react-components/react-dialog/src/components/DialogSurface/renderDialogSurface.tsx

```diff
@@ -1,25 +1,20 @@
 /** @jsxRuntime classic */
-/** @jsx createElement */
+/** @jsx createElementNext */
 
-import { createElement } from '@fluentui/react-jsx-runtime';
+import { createElementNext } from '@fluentui/react-jsx-runtime';
```

Comment:
It's draft, so it's okay to use `createElementNext`, but for a final change let's consider better naming